src/instance/exports.rs

```diff
@@ -85,6 +85,78 @@ impl InspectExportedFunction for ExportedFunction {
     }
 }
 
+pub(super) fn call_dyn_func(
+    py: Python,
+    function_name_as_str: &str,
+    function: DynFunc,
+    arguments: &PyTuple,
+) -> PyResult<PyObject> {
+    // Check the given arguments match the exported function signature.
+    let signature = function.signature();
+    let parameters = signature.params();
+
+    let number_of_parameters = parameters.len() as isize;
+    let number_of_arguments = arguments.len() as isize;
+    let diff: isize = number_of_parameters - number_of_arguments;
+
+    match diff.cmp(&0) {
```

Comment:
Why a match instead of if/else ? (that might be more readable)

Reply:
It makes Clippy happy.